etcdctl/ctlv2/command/util.go

```diff
@@ -167,16 +170,24 @@ func getTransport(c *cli.Context) (*http.Transport, error) {
 	if keyfile == "" {
 		keyfile = os.Getenv("ETCDCTL_KEY_FILE")
 	}
+	if insecureSkipVerify == false {
+		insecureSkipVerify = os.Getenv("ETCDCTL_INSECURE_SKIP_TLS_VERIFY") != ""
```

Comment:
I think this should be something to the effect of  `insecureSkipVerify = strings.EqualFold(os.Getenv("ETCDCTL_INSECURE_SKIP_TLS_VERIFY"), "TRUE")`. Anything that doesn't match "true" from the ENV VAR is considered implicitly false.